commit 7d7f8f8e2c2c5b2321af8422adb858fee4d3798e (refs/changes/93/4393/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-06-21T12:47:56-07:00 (1 year, 4 months ago)
    
    TRITON-530 nightly-1 vmapi fabric provision test is failing on "should have found one NAT vm"
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Approved by: Dave Eddy <dave.eddy@joyent.com>

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index 01e11b3..243fb6e 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2798,8 +2798,10 @@ exports.destroy_test_vms_final = function (t) {
 exports.find_fabric_network = function (t) {
     assert.arrayOfObject(NETWORKS, 'NETWORKS');
 
+    // Find a fabric network that is owned by CUSTOMER.
     fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
-        return n.fabric === true;
+        return n.fabric === true && Array.isArray(n.owner_uuids) &&
+            n.owner_uuids.indexOf(CUSTOMER) >= 0;
     });
 
     t.ok(fabricNetwork, 'found a fabric network');
